packages/commonwealth/client/scripts/controllers/server/newProfiles.ts

```diff
@@ -13,22 +13,22 @@ class NewProfilesController {
     return this._store;
   }
 
-  private _unfetched: Profile[];
+  private _unfetched: Map<string, Profile>;
 
   private _fetchNewProfiles;
 
   public allLoaded() {
-    return this._unfetched.length === 0;
+    return this._unfetched.size === 0;
   }
 
   public isFetched = new EventEmitter();
 
   public constructor() {
-    this._unfetched = [];
+    this._unfetched = new Map();
     this._fetchNewProfiles = _.debounce(() => {
-      this._refreshProfiles(this._unfetched);
-      this._unfetched.splice(0);
-    }, 50);
+      console.log([...this._unfetched.values()].length);
```

Comment:
do we need console log here?

Reply:
Here is some explanation about this:
https://commonxyz.slack.com/archives/C050AE0URFH/p1681940005881929

Bad addresses in DB - stopped last year - 27 such addresses, these are the ones which produce this ugly assertion error for Ethereum addresses
```
SELECT * FROM "Addresses" where address LIKE '%sleep%'
```